addons/event_sale/models/account_move.py

```diff
@@ -10,7 +12,38 @@ class AccountMove(models.Model):
     def _invoice_paid_hook(self):
         """ When an invoice linked to a sales order selling registrations is
         paid confirm attendees. Attendees should indeed not be confirmed before
-        full payment. """
+        full payment. Notify the partners with their tickets."""
         res = super(AccountMove, self)._invoice_paid_hook()
-        self.mapped('line_ids.sale_line_ids')._update_registrations(confirm=True, mark_as_paid=True)
+        sale_order_line_ids = self.mapped('line_ids.sale_line_ids')
+        sale_order_line_ids._update_registrations(confirm=True, mark_as_paid=True)
+
+        values = self.env['event.registration'].sudo()._read_group(
```

Comment:
That's not the purpose of account.move to trigger business code on registration model ... especially we have an automated mail scheduler mechanism on events. Why would we need that ?

Reply:
We could add a new 'interval_type' on 'event.type.mail' model in 'event_sale': after_pay, which would be trigerred once tickets are noted as paid. There is already an "after_sub" when you register, this one would be used once paid. Maybe it could do the trick (and then be configurable / event). What do you think ?

Reply:
Actually after discussing with Jija, we can completely remove this as it is redundant with the checkout at the end of the ticket booking flow. :)